base_changeset/security/groups.xml

```diff
@@ -17,9 +17,9 @@
     <data noupdate="1">
         <record id="group_changeset_manager" model="res.groups">
             <field name="users" eval="[(4, ref('base.user_root'))]" />
+            <field name="implied_ids" eval="[(4, ref('group_changeset_manager'))]" />
         </record>
         <record id="group_changeset_user" model="res.groups">
-            <field name="implied_ids" eval="[(4, ref('group_changeset_manager'))]" />
```

Comment:
this I think was just a mixup, can you confirm @astirpe?

Reply:
Yes I confirm! (and strange that nobody noticed it yet!)

But I would keep the:

```xml
        <record id="group_changeset_manager" model="res.groups">
            <field name="users" eval="[(4, ref('base.user_root'))]" />
        </record>
```